File: Python/백준/Bronze/21280. Förvirrad föreläsare/Förvirrad föreläsare.py
a = int(input())
aa = list(map(int,input().split()))
e = 0 #빈 강의실에서 수업할 때
m = 0 #강의를 놓쳤을 때
p = 0 #이전 주의 강의 수
for i in range(a):
    if i!=0:
        if (p>aa[i]):
            e += p-aa[i]
        else: 
            m += aa[i]-p
    p = aa[i]
print(e,m)

File: C#/백준/Bronze/21280. Förvirrad föreläsare/Förvirrad föreläsare.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int e = 0;
            int m = 0;
            int p = 0;
            for (int i=0;i<a;i++){
                if(i!=0){
                    if(p>aa[i]){
                        e+=p-aa[i];
                    }
                    else{
                        m+=aa[i]-p;
                    }
                }
                p = aa[i];
            }
            Console.WriteLine($"{e} {m}");
        }
    }
}